docs/flocker-features/pure-storage-configuration.rst

```diff
@@ -0,0 +1,13 @@
+.. _pure-storage-backend:
+
+=================================
+Pure Storage Block Device Backend
+=================================
+
+.. begin-body
+
+Pure Storage provides a Flocker backend plugin.
```

Comment:
change to "Pure Storage provides a Flocker backend plugin for FLASHARRAY//m All flash storage system."

Then add a setence at says "FlashArray//m enables you to transform your rack, data center, or cloud, with an afordable all-flash array capable of consolidating all your key business applications."